### ISSUE TITLE
'Use [...] instead' QuickFix shouldn't fix array adds (`list($foo[], $bar) = func();`)

### ISSUE THREAD

**Issue by HenkPoley** [kalessil/phpinspectionsea#1661]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) 4.0.6 |
| Language level | 7.x                        |

Current behaviour
--
```php
$foo = [];
$bar = [];
list($foo[], $bar) = func();
```

is 'fixed' to:

```php
$foo = [];
$bar = [];
[$foo[], $bar] = func();
```

Which as far as I can see, the behavior of list() is to add to the array $foo. While the square bracket notation doesn't understand this.

Expected behaviour
--

Leave this specific case alone.

Environment details
---

**Comment by jdreesen** [comment]
> While the square bracket notation doesn't understand this.

Yes it does, see: https://3v4l.org/chklF

The two are equivalent afaik (at least they should be).

**Comment by HenkPoley** [comment]
Ah cool, so PhpStorm is complaining for nothing 🙊

**Comment by jdreesen** [comment]
Oh. Seems like a bug in PhpStorm, then. Maybe you want to create a bug report in their issue tracker?

**Comment by HenkPoley** [comment]
There's already:

* https://youtrack.jetbrains.com/issue/WI-45906

**Comment by jdreesen** [comment]
It seems like this will be fixed in the next PhpStorm release.
